### DISCUSSION TITLE
Make the argon "pro" song progress component default to having difficulty graph hidden.

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#21814] [Ideas]
Just a small QoL change i think ""pro"" players will appreciate, the current full size implementation is nice, but a bit bulky and distracting for serious play IMO

**Comment by Alsen99** [comment]
I agree, and kind of unnecessary for pros because they remember which part of the song are difficult

**Comment by ItsShamed** [comment]
EDIT: Maybe late since release has already happened but has #22144 solved this for you?

I think the [Argon variant](https://www.figma.com/file/up6guVobzpc3DZlQxWzVmv/Gameplay-Layout?node-id=2%3A29&t=42l5vNscJiD4N4in-1) of the song progress bar would be a great solution to this. (although I'm not sure what to do with UX)

Update: the implementation is basically finished, but I'm not even sure if the designs are even *"final"*, so I'll wait until I get a "go for PR" signal from the core team.

https://user-images.githubusercontent.com/39100084/210313679-e2000646-6f28-4c6e-95c4-81a0196a0328.mp4

**Comment by peppy** [comment]
Thanks for your feedback.
